hack/ukify/measure/extend/extend.go

```diff
@@ -0,0 +1,41 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
we should move this to `internal/tpm2` or something when we move `ukify` into installer code